camel/dtaq_to_kafka/src/main/java/com/ibm/jesseg/MainApp.java

```diff
@@ -0,0 +1,44 @@
+package com.ibm.jesseg;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.impl.DefaultCamelContext;
+
+/**
+ * A Camel Application that routes messages from an IBM i data queue to a Kafka server.
+ */
+public class MainApp { 
+    public static void main(final String... args) throws Exception {
+
+        // This class simply reads values and creates URIs from the config.properties file.
+        // This is not a standard Camel class, but is part of this example. You can feel free
+        // to just write the URIs instead of using this class.
+        DtaQToKafkaConfig conf = new DtaQToKafkaConfig();
+
+        // Standard for a Camel deployment. Start by getting a CamelContext object.
+        CamelContext context = new DefaultCamelContext();
+        System.out.println("Apache Camel version "+context.getVersion());
+
+        // Now, it's pretty simple to define a Camel route!!
+        final String dtaqUri = conf.getDtaQUri(); //something like -> jt400://username:password@localhost/qsys.lib/mylib.lib/myq.DTAQ?keyed=false&format=binary&guiAvailable=false
+        final String smtpUri = conf.getKafkaUri(); //something like -> kafka:mytopic?brokers=mybroker:9092
+        context.addRoutes(new RouteBuilder() {
+            @Override
+            public void configure() {
+                from(dtaqUri)
+                .wireTap("log:msgq_to_email?showAll=true&level=INFO") // This is just for debugging data flowing through the route
+                .to(smtpUri); 
+            }
+        });
```

Comment:
Might want to give more detail here about what's going on. This is essentially the crux of the example and it's seemingly smooshed in with getting some URIs.

Reply:
Good point. I'll write up some info here. I had initially thought about walking through the snippet in the main README but ended up forgetting completely.